tests/integration/encryption/peer_acp_test.go

```diff
@@ -220,6 +220,23 @@ func TestDocEncryptionACP_IfUserHasAccessButNotNode_ShouldNotFetch(t *testing.T)
 					"Users": []map[string]any{},
 				},
 			},
+			// If the instance doesn't have rights to the doc, it can't do block sync
+			// and therefore has no heads. So commits should be empty.
```

Comment:
suggestion: `... and therefore doesn't have the related commit blocks`. I think we had discussed this previously but saying it `has no heads` isn't the reason why `commits` returns empty.